contracts/TokenRequest.sol

```diff
@@ -201,9 +201,29 @@ contract TokenRequest is AragonApp {
 
         tokenManager.mint(requesterAddress, requestAmount);
 
-        addressesTokenRequestIds[requesterAddress].deleteItem(_tokenRequestId);
+        addressesTokenRequestIds[msg.sender].deleteItem(_tokenRequestId);
```

Comment:
Why have you changed this? This should be the `requesterAddress` so it deletes the right token request.

Reply:
ughh that was something with the merge, i didn't change it